addons/mail/static/src/model/model_field.js

```diff
@@ -124,7 +124,15 @@ export class ModelField {
          * the first operation that is able to determine an order for the two
          * records.
          */
-        this.sort = sort;
+        this.sort;
```

Comment:
I don't think this one is still necessary (or it can be kept to contain the new value, but we probably don't need an extra var).